testing/sdk_metrics.py

```diff
@@ -24,6 +25,22 @@ def get_scheduler_metrics(service_name, timeout_seconds=15 * 60):
     return sdk_cmd.service_request("GET", service_name, "/v1/metrics").json()
 
 
+def get_failure_metrics(service_name: str) -> typing.Dict:
+    history = sdk_cmd.history_request().json()
```

Comment:
Should we call `resp.raise_for_status()` here?

Reply:
The `.json()` raises an exception on `204` or invalid JSON types. What are additional checks you think of?

Reply:
This is a HTTP request, and can have any number of return values that indicate some issue. Using `raise_for_status()` just helps to show us that these errors occurred instead of having to debug the reason for the `.json()` call failing.